dts/riscv/gigadevice/gd32vf103.dtsi

```diff
@@ -40,7 +40,7 @@
 		systimer: timer@d1000000 {
 			compatible = "nuclei,systimer";
 			reg = <0xd1000000 0x10000>;
-			interrupts = <3 0>, <7 0>;
+			interrupts = <7 0>;
```

Comment:
As far as I know, IRQ 3 is part of the system timer (SysTimerSW_IRQn).

Reply:
I agree with @gmarull here https://doc.nucleisys.com/nuclei_spec/isa/timer.html

Reply:
More refer from section 6 of [Bumblebee Processor Core Instruction Set Architecture Manua](https://raw.githubusercontent.com/nucleisys/Bumblebee_Core_Doc/master/Bumblebee%20Core%20Architecture%20Manual.pdf)

Reply:
Hi, @gmarull, @carlocaione, @cameled 

Thank you for your comments. I was a bit harry fixing the timer not working.
I should have thought a little more.

As you are pointing out, msip is part of the TIMER unit in GD32VF103.
So, It should be left there.

This PR intends to restore previously working behavior.

That behavior is,
- RISC-V machine timer uses IRQ7.
- IRQ3 is not handled.

It can by changing the order of interrupts.

I think it should be better to create another PR if changing to handle IRQ3.
IRQ3/msip seems to use to Inter Processor Interruption on SMP.
I think good to do nothing because GD32VF103 is a single core.